central/scannerdefinitions/handler/handler.go

```diff
@@ -109,6 +119,14 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *httpHandler) get(w http.ResponseWriter, r *http.Request) {
+	fileType := r.URL.Query().Get("file")
```

Comment:
This looks to be a new query string parameter, are we intentionally using `file` for backwards compatibility? or can this be renamed to `type` (or similar)?

Reply:
**type** should work. I just couldn't think of any better parameter name when I was making this change

Reply:
Gotcha, the reason I asked is because `file` is used below in the same function, seemed to be serving a dual purpose.